test/CappedCrowdsale.js

```diff
@@ -62,7 +70,7 @@ contract('CappedCrowdsale', function ([_, wallet]) {
   describe('ending', function () {
 
     beforeEach(async function () {
-      await advanceToBlock(this.startBlock - 1)
+      await increaseTime(moment.duration(1, 'week'))
```

Comment:
I totally see your point, as I've got exactly the same idea at the beginning. 

The problem is that testrpc exposes only `evm_increaseTime` method in the API. There are 3 risks that may cause `flaky tests` if we decide to implement `increaseTimeTo()` on top of `evm_increaseTime` by calculating the remaining time and to target point:

1. We need to call two methods `evm_increase`, `evm_mine` and there may be an additional delay between them.
2. We are using rpc calls which add additional delay. 
3. We do not have any control over `testrpc` rounding policy used to calculate block timestamps. 

As I totally agree that increasing time to a fixed timestamp is a better approach, we will need to have a native `testrpc` function to implement it correctly. I'm going to raise an issue for `testrpc` and look if it can be implemented easily.